CONTRIBUTING.md

```diff
@@ -138,22 +138,11 @@ Docker images that contains `dgraph` and `badger` commands.
 ### Testing
 
 #### Dgraph
-Run the `test.sh` script in the root folder.
-
-
-    $ ./test.sh
-    
-    INFO: Running tests using the default cluster
-    …
-    INFO: Running test for github.com/dgraph-io/dgraph/algo
-    ok  	github.com/dgraph-io/dgraph/algo	0.004s
-    INFO: Running test for github.com/dgraph-io/dgraph/codec
-    ok  	github.com/dgraph-io/dgraph/codec	9.308s
-    INFO: Running test for github.com/dgraph-io/dgraph/codec/benchmark
-    ?   	github.com/dgraph-io/dgraph/codec/benchmark	[no test files]
-    …
-
-Run `test.sh --help` for more info.
+1. Change directory to t directory. 
+2. If all packages need to be tested, run
+      make test
+   If only a specific package needs to be tested, run
+      make test args="--pkg=desired_package_name"
```

Comment:
could we add an example here? It may not be clear as too what -pkg should be set to. And I am wondering whether we should also specify how to use the `t` binary?

Reply:
can we clean up the makefile and reference the `make test` here?